add ECS support (#36)
logstash-plugins/logstash-codec-avro: docs/index.asciidoc

tag events with `_avroparsefailure` when decode fails

[id="plugins-{type}s-{plugin}-target"]
===== `target`

* Value type is <<string,string>>
* There is no default value for this setting.

Define the target field for placing the row values. If this setting is not
set, the Avro data will be stored at the root (top level) of the event.

> 

Author: this is important. many thanks

> For example, if you want data to be put under the `document` field:

Reviewer: ```suggestion
*Example* 

In this configuration, data will be placed under the `document` field.
```

[source,ruby]
----------------------------------
input {
  kafka {
    codec => avro {
        schema_uri => "/tmp/schema.avsc"
        target => "[document]"
    }
  }
}

> ----------------------------------

Reviewer: ```suggestion
----------------------------------

```
